src/modules/localStorage.js

```diff
@@ -0,0 +1,20 @@
+import { toDo } from '../index.js'; import { createListElement } from './newTask.js';
```

Comment:
I feel the index.js should not be exporting as it is the main file, rather it should be importing all the functions.